src/client/serialize/SConscript

```diff
@@ -0,0 +1,29 @@
+"""Build FUSE client"""
+import daos_build
+
+def scons():
+    """Execute build"""
+    Import('env')
+
+    denv = env.Clone()
+
+    have_hdf5 = True
+
+    #If hdf5 installed build hdf_serialize library
+    conf = Configure(denv)
+    if not conf.CheckLibWithHeader('hdf5', 'hdf5.h', 'c'):
+        have_hdf5 = False
+    conf.Finish()
+
+    libraries = ['daos', 'hdf5']
+
+    src = ['hdf_serialize.c']
+    if have_hdf5 is True:
+        print("Building with hdf5 bindings.")
+        hdf_serialize = daos_build.library(denv, 'hdf_serialize', src,
```

Comment:
i don't think hdf_serializer is a suitable name for this.
hdf is very common to many libraries. we are using HDF5 here. this is also a DAOS tool not an HDF5 one.
serialize is also very generic. we will support serializing just not the md part?
so we should be calling this something like daos_md_serialize or something like that?

Reply:
@mchaarawi yes this will support more than just the attribute part. It will also contain the hdf5 serialization code necessary for the daos serialize/deserialize tools. What about naming it just daos_serialize?

Reply:
im fine with whatever as long as you don't call it hdf